src/composables/sdk13.ts

```diff
@@ -182,10 +201,15 @@ export function useSdk13({ store }: IDefaultComposableOptions) {
   );
 
   return {
+    nodeNetworkId,
     isNodeReady,
     isNodeConnecting,
     isNodeError,
     isSdkReady,
+    isNodeMainnet,
+    isNodeTestnet,
```

Comment:
I suggest to move these variables to have a group where names starts with `isNode...` :)